hasjob/static/js/styles-app.js

```diff
@@ -0,0 +1 @@
+import '../sass/app.sass';
```

Comment:
If this needs compilation, it should be in the `assets` folder, not `static`.

Reply:
Everything in `static` is published to the world and can be directly accessed from a URL.

Reply:
Fixed